webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3970,7 +3970,7 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                 KommunicateUI.updateLeadCollectionStatus(err, message, data.message || []);
             };
 
-            _this.loadMessageList = function (params, callback) {
+            _this.loadMessageList = function (params, callback,isWaitingQueue) {
```

Comment:
@Kusum-Dhoundiyal pass "isWaitingQueue" value in params itself, no need to create new function variable.